src/main/java/UserAnalyzer.java

```diff
@@ -0,0 +1,20 @@
+/*
+This file contains UserAnalyzer class, which is the parent class for all analyzers
+ */
+package main.java;
+
+public class UserAnalyzer implements UserInformation{
```

Comment:
If UserAnalyzer is a superclass for the other analyzers, shouldn't this be an abstract class instead? If not, will we instantiate this object in the future?